website/docs/backends/types/swift.html.md

```diff
@@ -19,14 +19,13 @@ Stores the state as an artifact in [Swift](http://docs.openstack.org/developer/s
 ```hcl
 terraform {
   backend "swift" {
-    path = "terraform-state"
+    path = "terraform-state",
+    state_name = "terraform.tfstate"
   }
 }
 ```
 This will create a container called `terraform-state` and an object within that container called `tfstate.tf`.
```

Comment:
Need to update this line here as well since you changed the example.

Reply:
Well spotted! I've made a commit to fix the discrepancies. Basically, the old code was hardwired to use "tfstate.tf". Unfortunately, the documentation referred to "terraform.tfstate" in many places. The new commit fixes the docs to reflect the reality that the state has always been "tfstate.tf"